src/server/pkg/worker/worker_service.proto

```diff
@@ -13,16 +16,19 @@ message ProcessRequest {
   repeated pfs.FileInfo data = 1;
 }
 
-// ProcessResponse contains either a tag, if the processing was successful,
-// or a log that's the combination of stdout+stderr, if the processing
-// was unsuccessful.
-// TODO: allow for capturing logs even if processing was successful.
+// ProcessResponse contains a tag, only if the processing was successful.
 message ProcessResponse {
   pfs.Tag tag = 1;
   // If true, the user program has errored
   bool failed = 2;
 }
 
+message CancelRequest {
+  repeated string data_filters = 1;
+}
+
 service Worker {
   rpc Process(ProcessRequest) returns (ProcessResponse) {}
+  rpc Status(google.protobuf.Empty) returns (pps.WorkerStatus) {}
```

Comment:
Same thing here; ideally we'd have `StatusRequest` even if its empty, for future extensiblility.  No need to fix it rn tho.